### PR TITLE
enable push rule '.m.rule.master' to disable all pushnotification

### PR DESCRIPTION
**Pull request by fredo**
Disable all push notifications for a given user upon first login by enabling push rule `.m.rule.master`

Fixes: #6783

### CODE REVIEW

File: raiden/network/transport/matrix/client.py [hunk 1]
```diff
@@ -245,6 +245,13 @@ def get_aliases(self, room_id: str) -> Dict[str, Any]:
     def __repr__(self) -> str:
         return f"<GMatrixHttpApi base_url={self.base_url}>"
 
+    def disable_push_notifications(self) -> Dict[str, Any]:
+        return self._send(
+            "PUT",
+            "/pushrules/global/override/.m.master.rule/enabled/",
```
Comment (konradkonrad) [inline raiden/network/transport/matrix/client.py:251]: Looks like a typo?
```suggestion
            "/pushrules/global/override/m.master.rule/enabled/",
```

Comment (fredo) [reply raiden/network/transport/matrix/client.py:251]: looks like but it's not ;) server-default push rules start with a `.`. Here is the [spec](https://matrix.org/docs/spec/client_server/r0.6.0#predefined-rules)

**Review by konradkonrad — APPROVED** [review]
please check for the API path used, is it a typo or is the leading `.` intentional?
Other than that looks good to me!

**Comment by fredo** [comment]
see above

**Pull request merged**